src/Ether.Vsts/Handlers/Commands/DeleteWorkItemsHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Ether.Contracts.Interfaces;
+using Ether.Contracts.Interfaces.CQS;
+using Ether.Vsts.Commands;
+using Ether.Vsts.Dto;
+using static Ether.Contracts.Types.NullUtil;
+
+namespace Ether.Vsts.Handlers.Commands
+{
+    public class DeleteWorkItemsHandler : ICommandHandler<DeleteWorkItems>
+    {
+        private readonly IRepository _repository;
+
+        public DeleteWorkItemsHandler(IRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public async Task Handle(DeleteWorkItems command)
+        {
+            CheckIfArgumentNull(command.Ids, nameof(command.Ids));
+
+            if (!command.Ids.Any())
+            {
+                return;
+            }
+
+            foreach (var id in command.Ids)
+            {
+                await _repository.DeleteAsync<WorkItem>(workitem => workitem.WorkItemId == id);
+            }
+
+            var idsInStrings = command.Ids.Select(id => id.ToString());
+            var teamMembers = await _repository.GetByFilteredArrayAsync<TeamMember>("RelatedWorkItems", idsInStrings.ToArray());
```

Comment:
I assume you're trying to get all team members that are workitems from `command.Ids`.
Try:
```suggestion
            var teamMembers = await _repository.Get<TeamMember>(m => m.RelatedWorkItems.Any(w => command.Ids.Contains(w)) );
```

Reply:
Changed and tested, it works. Applied in the new comment.